packages/gcc.rb

```diff
@@ -22,6 +22,7 @@ class Gcc < Package
      x86_64: '01efdcdab6bebc32fd5608f6a061c7ceebd69e9f96084293a903f5af9c3576f3'
   })
 
+  depends_on 'binutils'
```

Comment:
Does this require `binutils` in core.rb?

Reply:
I thought about it, but I decided against it. Whether binutils should be in core.rb independently of this is something else altogether, and I don't have any strong opinions on that either way.

Reply:
Since this gcc is a core package this dependency will get inherited so I think it should be added.  Also, it would be nice to update `tools/core_packages.txt` too.